tutorials/bios-boot-tutorial/bios-boot-tutorial.py

```diff
@@ -293,9 +296,56 @@ def stepCreateTokens():
     run(args.cleos + 'push action eosio.token issue \'["eosio", "%s", "memo"]\' -p eosio' % intToCurrency(totalAllocation))
     sleep(1)
 def stepSetSystemContract():
-    retry(args.cleos + 'set contract eosio ' + args.contracts_dir + '/eosio.system/')
+    # All of the protocol upgrade features introduced in v1.8 first require a special protocol 
+    # feature (codename PREACTIVATE_FEATURE) to be activated and for an updated version of the system 
+    # contract that makes use of the functionality introduced by that feature to be deployed. 
+
+    # activate PREACTIVATE_FEATURE before installing eosio.boot
+    retry('curl -X POST http://127.0.0.1:%d' % args.http_port + 
+        '/v1/producer/schedule_protocol_feature_activations ' +
+        '-d \'{"protocol_features_to_activate": ["0ec7e080177b2c02b278d5088611686b49d739925a92d9bfcacd7fc6b74053bd"]}\'')
+    sleep(3)
+
+    # install eosio.boot which supports the native actions and activate 
+    # action that allows activating desired protocol features prior to 
+    # deploying a system contract with more features such as eosio.bios 
+    # or eosio.system
+    retry(args.cleos + 'set contract eosio ' + args.contracts_dir + '/eosio.boot/')
+    sleep(3)
+
+    # activate remaining features
+    # GET_SENDER
+    retry(args.cleos + 'push action eosio activate \'["f0af56d2c5a48d60a4a5b5c903edfb7db3a736a94ed589d0b797df33ff9d3e1d"]\' -p eosio')
+    # FORWARD_SETCODE
+    retry(args.cleos + 'push action eosio activate \'["2652f5f96006294109b3dd0bbde63693f55324af452b799ee137a81a905eed25"]\' -p eosio')
+    # ONLY_BILL_FIRST_AUTHORIZER
+    retry(args.cleos + 'push action eosio activate \'["8ba52fe7a3956c5cd3a656a3174b931d3bb2abb45578befc59f283ecd816a405"]\' -p eosio')
```

Comment:
For consistency can we make these all `eosio@active`

Reply:
Will change. Thanks.